ui/jobs/jobs.js

```diff
@@ -2689,6 +2695,57 @@ class Bars {
     }
   }
 
+  OnCraftingLog(log) {
+    // Hide CP Bar when not crafting
+    const container = document.getElementById('jobs-container');
+
+    const anyRegexMatched = (line, array) => {
+      for (const regex of array) {
+        if (regex.test(line))
+          return true;
+      }
+      return false;
+    };
+    const undefinedOrEqual = (value, expected) => {
```

Comment:
This is only used once, so maybe doesn't need to be a function?

Reply:
Sure, I agree too.